File: ServiceBus/MessagingExpress/Topic/receive-message.js
const { ServiceBusClient } = require("@azure/service-bus");

const connectionString = ""
const serviceBusClient = new ServiceBusClient(connectionString);

async function receiveMessages(topicName, subscriptionName) {
    try
    {
        // create receiver for topic subscription
        const receiver = serviceBusClient.createReceiver(topicName, subscriptionName);
        // register handlers for message or error events
        receiver.subscribe({
            // called whenever a message is received
            processMessage: async (message) => {
                // do something with message
                console.log(message);

                // complete the message to remove it from subscription
                await receiver.completeMessage(message);
            },
            processError: async (err) => {
                console.log(err);
            }
        });

        await receiver.close();
    }
    catch(err){
        console.log(err);
    }
    finally {
        await serviceBusClient.close();
    }
}

await receiveMessages("topic001", "subs001");

File: ServiceBus/MessagingPython/Topic/receive-message.py
from azure.servicebus import ServiceBusClient

connection_str = ""
topic_name = "topic001"
subscription_name = "subs001"

def receive_messages(connection_str, topic_name, subscription_name):
    servicebus_client = ServiceBusClient.from_connection_string(connection_str)
    receiver = servicebus_client.get_subscription_receiver(topic_name, subscription_name)
    with receiver:
        for message in receiver:
            # Do something with the message
            print(message)
            receiver.complete_message(message)
    receiver.close()
    servicebus_client.close()

receive_messages(connection_str, topic_name, subscription_name)
